include/call.hpp

```diff
@@ -322,7 +322,7 @@ class call : virtual public task, virtual public listener, public virtual socket
     BIO *m_bio;
 #endif
 
-    int _callDebug(const char *fmt, ...);
+    int _callDebug(const char *fmt, ...) __attribute__((format(printf, 2, 0)));
```

Comment:
Should this be `2, 3` because of the hidden this pointer? Or should it be `1, 2`?